README.md

```diff
@@ -27,6 +27,18 @@ Whilst working on this repo, it is advised that you review your own changes loca
 
 Please make sure you fork the repo and change the clone URL in the example below for your fork:
 
+- Linux Mint / Ubuntu 18.04 LTS (using `python`):
+   - Preparations (only required once):
+   ```bash
+   git clone https://github.com/YOUR-USERNAME/docs
+   cd docs
+   sudo apt install python3-pip
```

Comment:
This should be adapted like it's in master to use the requirements file IMHO

Reply:
This isn't resolved.

Reply:
Ah, sorry. I forgot to add `README.md` in my last commit. It'll be resolved now.

Reply:
Weird, I still don't see this fixed in this branch.